doc/nrf/known_issues.rst

```diff
@@ -1090,7 +1150,7 @@ KRKNWK-6255: RSSI parameter adjustment is not applied
 
     **Workaround:** To apply RSSI parameter adjustments, cherry-pick the commits in `hal_nordic PR #88 <https://github.com/zephyrproject-rtos/hal_nordic/pull/88>`_, `sdk-nrfxlib PR #381 <https://github.com/nrfconnect/sdk-nrfxlib/pull/381>`_, and `sdk-zephyr PR #430 <https://github.com/nrfconnect/sdk-zephyr/pull/430>`_.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
 
```

Comment:
```suggestion

.. rst-class:: v1-7-0

KRKNWK-11384: At the rare occasions during the use of BLE and 802.15.4 multiprotocol the device might assert.

KRKNWK-11204: Transmitting the 802.15.4 frame with the improperly populated Auxilliary Security Header might result in assert.
```

Reply:
11384 is added.
11204 is closed, which poses a question if you're planning to update it. If not, it is a limitation and should go into the changelog in sdk-nrfxlib. If yes, it requires a new JIRA.

Reply:
Checked with @czeslawmakarski . The second issue is a limitation and will be added to https://github.com/nrfconnect/sdk-nrfxlib/pull/549